src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -51,6 +52,9 @@ public OI(Joystick driverStick, Joystick controlStick) {
 
         // Joystick for controlling the mechanisms of the robot
         this.controlStick = controlStick;
+        
+        EJoystickButton CameraButton = new EJoystickButton(driverStick, 3);
```

Comment:
Since this is using the `driverStick` try to put this code under the correct joystick.